run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
This already looks fishy. Use `#/usr/bin/env bash`.

Reply:
Fair, I wasn't thinking much and just used bin/bash because I'm used to doing it the quick and dirty way.